pkg/action/diff.go

```diff
@@ -230,8 +244,10 @@ func fileMove(ctx context.Context, c bincapz.Config, fr, tr *bincapz.FileReport,
 		}
 	}
 
-	// Move these into the modified list.
-	d.Modified[apath] = abs
-	delete(d.Removed, rpath)
-	delete(d.Added, apath)
+	// Move these into the modified list if the files are not completely different (something like ~0.3)
+	if score > 0.3 {
```

Comment:
I added this because running something like this makes bincapz think the original binary was changed:
```sh
$ go run . --format=simple --diff samples/Linux/clean/ls.x86_64 samples/macOS/2024.LightSpy/dropper
```